includes/class-llms-blocks-assets.php

```diff
@@ -55,8 +55,15 @@ public function editor_assets() {
 			true
 		);
 
-		$i18n_dir = defined( 'LLMS_BLOCKS_LIB' ) && LLMS_BLOCKS_LIB ? LLMS_PLUGIN_DIR . '/languages' : LLMS_BLOCKS_PLUGIN_DIR . '/i18n';
-		wp_set_script_translations( 'llms-blocks-editor', 'lifterlms', $i18n_dir );
+		$l10n_dir   = 'i18n';
+		$plugin_dir = LLMS_BLOCKS_PLUGIN_DIR;
+
+		// If the plugin is being loaded as a library, use the files included in the core plugin.
+		if ( defined( 'LLMS_BLOCKS_LIB' ) && LLMS_BLOCKS_LIB ) {
+			$l10n_dir   = null;
+			$plugin_dir = null;
+		}
+		llms_set_script_translations( 'llms-blocks-editor', 'lifterlms', $plugin_dir, $l10n_dir );
```

Comment:
the LifterLMS core safe directory (filterable) always takes the precedence, should we add a comment here, for us? :D

Reply:
@eri-trabiccolo good call, hows this comment?

Reply:
@thomasplevy <3